engine/runtime/src/bench/scala/org/enso/interpreter/bench/IRUtils.scala

```diff
@@ -0,0 +1,46 @@
+package org.enso.interpreter.bench
+
+import org.enso.compiler.core.IR
+
+object IRUtils {
+  private def collectBenchCalls(ir: IR.Module): List[BenchCall] = {
+    val methods = ir.bindings.collect { case m : IR.Module.Scope.Definition.Method => m }
```

Comment:
Btw. I will need the JMH benchmarking for #5067...

I don't like mangling with `IR` much. We should change the `Bench.measure` API to be more JMH friendly instead.

Reply:
Yes, I agree that IR manipulation is too clumsy. Let's roll out with your *builder pattern* suggestion. It should not be difficult to collect the benchmark specification in the Java code that invokes JMH with that.